### PR TITLE
🍀 Support for before and after command.

### PR DESCRIPTION
**Pull request by gopinath-langote**
Issue: #1

Sample file:

```yaml
project: JUnit5
before: source "$HOME/.sdkman/bin/sdkman-init.sh" && sdk use java 11.0.2-open
after: source "$HOME/.sdkman/bin/sdkman-init.sh" && sdk use java 8.0.181-zulu
commands:
  - test: ./gradlew test
  - run: ./gradlew clean build
  - lint: ./gradlew spotlessApply
```

-----

The output of executing `1build lint` - 
```
\--------------------------------------------------\
Name: lint
Command: ./gradlew spotlessApply
Before: source "$HOME/.sdkman/bin/sdkman-init.sh" && sdk use java 11.0.2-open
After: source "$HOME/.sdkman/bin/sdkman-init.sh" && sdk use java 8.0.181-zulu
\--------------------------------------------------\

-e Using java version 11.0.2-open in this shell.

/> Configure project :
[versioning] WARNING - the working copy has unstaged or uncommitted changes.

BUILD SUCCESSFUL in 3s
38 actionable tasks: 38 up-to-date

-e Using java version 8.0.181-zulu in this shell.
```

### CODE REVIEW

**Comment by gopinath-langote** [comment]
@landpro Please review the same!

File: 1build [hunk 1]
```diff
@@ -59,17 +61,25 @@ def run(build_file_name, arguments):
...
 
 
-def execute(command):
+def execute(command, before=None, after=None):
+    cmd = command.cmd
+
     print(dash)
     print("Name: " + command.name)
     print("Command: " + command.cmd)
+    if before is not None:
```
Comment (eamanu) [inline 1build]: I think this could be write ```if before:``` ?

File: 1build [hunk 2]
```diff
@@ -59,17 +61,25 @@ def run(build_file_name, arguments):
...
+        print("Before: " + before)
+        cmd = before + "&&" + cmd
```
Comment (eamanu) [inline 1build]: Maybe this should be write something like: ```before + " && " + cmd``` ?

File: 1build (hunk 1 above)
Comment (gopinath-langote) [reply 1build]: @eamanu Thanks for pointing out the issue.

Updating PR.

File: 1build (hunk 2 above)
Comment (gopinath-langote) [reply 1build]: Make Sense!

**Comment by gopinath-langote** [comment]
@eamanu please feel free to add issues/PR for the project.

**Pull request merged**